### DISCUSSION TITLE
Licensing

### DISCUSSION THREAD

**Discussion by waldingl** [nathanwoulfe/Plumber-2#98] [General]
Hi @nathanwoulfe

We are pretty convinced that Plumber 2 is the right fit for our organisation but the trial limitation doesn't work for us in terms of trialling this effectively (this would involve multiple teams across the org).

Is there any other method to trial briefly for a domain and subdomain prior to purchase?

Thanks

**Comment by nathanwoulfe** [comment]
Hi @waldingl, have you tried with the test license (adding an empty `test.lic` file to app_plugins/plumber)? 

What exactly are you needing to verify?

**Reply by waldingl** [reply]
Hi @nathanwoulfe 

We need to determine if the marketing/compliance sign-off procedure within the org is satisfied with Plumber's features.

I read through the readme and noted that adding the empty `test.lic` file would restrict us to using naked localhost or localtest or domains ending in `.local` or `.test` - I was hoping to trial this in our staging environment which is currently mapped to a subdomain which is publicly accessible.

**Reply by nathanwoulfe** [reply]
Hi @waldingl, more than happy to answer any queries to help clarify for your team, but there's not really any way to license a non-local domain without buying the license, as the only difference between the test and paid license is restricting the former to local sites.
